### ISSUE TITLE
Accessibility Testing on Localhost

### ISSUE THREAD

**Issue by mcarbonell1** [microsoft/accessibility-insights-web#6166]
## Is your feature request related to a problem? Please describe.
Unable to run Accessibility Insights for Web on localhost. When I try to it directs me that: Accessibility Insights for Web can't run on this URL. You'll need to go to a different page.

## Describe the desired outcome
It would be really helpful from a testing stand point if one could run accessibility insights on something that is running on localhost. For example, if I wanted to make changes before pushing to ADO/Github and waiting for a build, I could locally test it and run Accessibility Insight for Webs.

## Describe alternatives you've considered
Pushing changes up and waiting for a build so I can test out a build link.

**Comment by mcarbonell1** [comment]
I realize this was the way I was launching the browser which made it unable to work. There was no URL for Accessibility Insights for Web to grab since I was launching Edge directly with the URL. Once I put in the localhost URL Accessibility Insights for Web worked.
